CMakeLists.txt

```diff
@@ -1574,6 +1607,17 @@ if(APPLE)
   endif()
 endif()
 
+target_link_libraries(strawberry PUBLIC
+    strawberry_lib
+    lib-networkremote
+)
+
+target_link_libraries(strawberry PRIVATE Qt6::Protobuf)
+
+if(HAVE_DISCORD_RPC)
+  target_link_libraries(strawberry_lib PRIVATE discord-rpc)
```

Comment:
This is already linked here: https://github.com/strawberrymusicplayer/strawberry/blob/3ab91ce090f0548b95511c15d07834468285213a/CMakeLists.txt#L1591

Reply:
Hmm, not a change I made I hope